package.json

```diff
@@ -50,14 +50,12 @@
   "spm": {
     "main": "d3.js"
   },
-  "dependencies": {
-    "jsdom": "1.0.0"
-  },
   "devDependencies": {
-    "smash": "~0.0.12",
-    "uglify-js": "2.4.0",
-    "vows": "0.7.0",
-    "seedrandom": "2.3.1"
+    "jsdom": "3",
+    "seedrandom": "2",
+    "smash": "0.0",
+    "uglify-js": "2.4",
```

Comment:
Would it be better to pin to a specific UglifyJS patch version, since changes in the patch version can cause d3.min.js to change?  For example, I had to update to 2.4.16 to get d3.min.js to match the build in this pull request.  On the other hand, not pinning to a patch version means less maintenance. :)